src/index.html

```diff
@@ -13,5 +13,151 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+          onsubmit="onSubmit()">
+      <div class="form-field" id="pers">
+
+        <fieldset class="form_regis">
+          <legend>Personal information
+          </legend>
+          <div class="form-field" id="registracia">
+      Username:
+            <input type="text"
+             name="username"
+             id="username">
+          </div>
+          <div class="form-field" id="name1">
+      Name:
+            <input type="text"
+             name="name"
+             id="name">
+          </div>
+          <div class="form-field" id="oldare1">
+      How old are You?
+            <input type="number"
+             name="averdg"
+             value="12"
+             maxlength="100"
+            id="oldare">
+          </div>
+          <div class="form-field" id="birdth1">
+        Full date of birth:
+            <input type="date"
+             id="birth"
+             name="daybirdth">
+          </div>
+          <div class="form-field" id="consent">
+        I accept the term of the agreement
+            <label class="form-field" for="agree">
+              <input type="checkbox"
+               name="term"
+               id="agree">
+            </label>
+          </div>
+        </fieldset>
+      </div>
+      <div class="form-fild" id="regis">
+        <fieldset class="form_regis">
+
+          <legend>Registration</legend>
+
+          <div class="form-field" id="email1">
+          E-mail:
+            <input type="email"
+                 name="email"
+                 value="email@example.com"
+                 id="email">
```

Comment:
```suggestion
            <input 
              type="email"
              name="email"
              value="email@example.com"
              id="email"
            >
```
Теги с большим количеством атрибутов нужно располагать вот так